.github/workflows/reusable_provenance.yaml

```diff
@@ -24,7 +24,10 @@ on:
 jobs:
   get_inputs:
     outputs:
-      artifact-path: ${{ steps.artifact-path.outputs.name }}
+      # Resolves to one or more files which are passed to the SLSA provenance generator.
+      artifact-path: ${{ steps.parse-build-config.outputs.artifact-path }}
+      # The name of the TR package. Coincides with the name of the buildconfig.
```

Comment:
Does `Coincides with the name of the buildconfig.` mean that we do that by convention, or that something will break if it is not the same? If the second, perhaps make the comment clearer that it must always be the same.

Reply:
The latter, i.e. it will break. Using stronger phrasing.